Tensile/KernelWriter.py

```diff
@@ -3596,7 +3592,10 @@ def getReplacementKernelPath(self, kernel):
     kernelName = self.getKernelName(kernel)
     return ReplacementKernels.Get(kernelName)
 
-  def shortenFileBase(self, base):
+  def shortenFileBase(self, kernel):
```

Comment:
This function's name no longer makes sense.  It should still just shorten the input.  The serial name should be implemented in `getKernelFileBase()` or in a function called from there.

Reply:
move ShortNames to getKernelFileBase()